_pages/dev/connecting-to-other-systems.md

```diff
@@ -136,6 +124,14 @@ backend: {
 }
 ```
 
-The OCC configuration is used in the `OccEndpointsService`. The service looks up the configuration and applies parameters to the endpoint if needed.
+The OCC configuration is used in the `OccEndpointsService`. The service looks up the configuration, and applies parameters to the endpoint, if needed.
 
-*Note*: The enpoints are typesafe, therefore the list of available endpoints is visible when adding the configuration.
\ No newline at end of file
+**Note:** The endpoints are type safe. As a result, the list of available endpoints is visible when adding the configuration.
+
+**Note:** This feature requires a feature level of 1.1 or higher. The following configuration is required to enable it:
```

Comment:
> This feature requires a feature level of 1.1 or higher
Can we already note that this is only true during release 1.? As soon as we enter > 2.0, we don't need the flag anymore.

Reply:
This will be true for many such statements throughout the docs. When we move to 2.0, I will need to remove these statements from our 2.0 docs (we will still have a site set up for people who want to access our 1.x docs). I have created a ticket to follow up for when we move to 2.0: https://github.com/SAP/cloud-commerce-spartacus-storefront-docs/issues/491